cves/CVE-2015-1831.yml

```diff
@@ -173,11 +175,11 @@ lessons:
     applies:
     note:
   native_wrappers:
-    applies:
-    note:
+    applies: true
+    note: It was because of a funtion that overrode the DefaultExcludedPatternsChecker that made the the hole appear.
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: This issue arrose because of not enough attention to this detail. They never considered certain input and left holes attackers could use.
```

Comment:
Considering "certain input" isn't exactly the problem here, the issue was caused by an overwrite of the functionality that DID consider that input.

Reply:
The description has been changed to better reflect my reasoning.